ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/HighAvailabilityTestUtil.java

```diff
@@ -352,6 +352,31 @@ static String extractQueryId(final String outputString) {
     };
   }
 
+  // Ensures that lags have been reported for the given host, but that the value is zero.
+  // Makes the simplified assumption that there's just one state store.
```

Comment:
im trying to think of a case where we might have more than one state store in a test, but i guess for `PullQueryLimitHARouting` we dont have any

Reply:
If we had multiple tables defined, it's possible.  But none of the tests which use this have that, so it's works fine.